ui/components/multichain/network-list-item/network-list-item.stories.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { NetworkListItem } from '.';
+
+export default {
+  title: 'Components/Multichain/NetworkListItem',
+  component: NetworkListItem,
+  argTypes: {
+    name: {
+      control: 'text',
+    },
+    selected: {
+      control: 'boolean',
+    },
+    onClick: {
+      action: 'onClick',
+    },
+    onDeleteClick: {
+      action: 'onClick',
+    },
+    iconSrc: {
+      action: 'text',
+    },
+  },
+  args: {
+    name: 'Ethereum',
+    iconSrc: '',
+    selected: false,
+    onClick: () => console.log('Network click!'),
+    onDeleteClick: () => console.log('Network delete!'),
```

Comment:
So by using actions it will show up here.

https://user-images.githubusercontent.com/8112138/226493056-4b9fa21e-eed0-4354-8bec-5786fa1cfad0.mov

But only if these are here
```suggestion
```

Need to do a storybook for engineers and will list this as a point